package.json

```
@@ -6,7 +6,7 @@
     "generate": "npm-run-all generate:*",
     "generate:payment-transactions-api": "rimraf generated/definitions/payment-transactions-api && shx mkdir -p generated/definitions/payment-transactions-api && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-functions-checkout/v1.1.2/openapi/index.yaml --no-strict --out-dir ./generated/definitions/payment-transactions-api --request-types --response-decoders --client",
     "generate:payment-ecommerce-v1": "rimraf generated/definitions/payment-ecommerce && shx mkdir -p generated/definitions/payment-ecommerce && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-infra/0f34ab7a924b5b309635d1fdd9fe58283ed91a8e/src/domains/ecommerce-app/api/ecommerce-checkout/v1/_openapi.json.tpl --no-strict --out-dir ./generated/definitions/payment-ecommerce --request-types --response-decoders --client",
-    "generate:payment-ecommerce-v2": "rimraf generated/definitions/payment-ecommerce-v2 && shx mkdir -p generated/definitions/payment-ecommerce-v2 && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-infra/main/src/domains/ecommerce-app/api/ecommerce-checkout/v2/_openapi.json.tpl --no-strict --out-dir ./generated/definitions/payment-ecommerce-v2 --request-types --response-decoders --client",
+    "generate:payment-ecommerce-v2": "rimraf generated/definitions/payment-ecommerce-v2 && shx mkdir -p generated/definitions/payment-ecommerce-v2 && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-infra/CHK-2798-openapi-multi-fee/src/domains/ecommerce-app/api/ecommerce-checkout/v2/_openapi.json.tpl --no-strict --out-dir ./generated/definitions/payment-ecommerce-v2 --request-types --response-decoders --client",
```

```suggestion
    "generate:payment-ecommerce-v2": "rimraf generated/definitions/payment-ecommerce-v2 && shx mkdir -p generated/definitions/payment-ecommerce-v2 && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-infra/main/src/domains/ecommerce-app/api/ecommerce-checkout/v2/_openapi.json.tpl --no-strict --out-dir ./generated/definitions/payment-ecommerce-v2 --request-types --response-decoders --client",
```
TODO: to be replaced after dependent infra pr merge